### DISCUSSION TITLE
Discounted NFT Mint Fees based on NFT Burn Count

### DISCUSSION THREAD

**Discussion by mshakeg** [hiero-ledger/hiero-improvement-proposals#674] [Ideas]
## Background
Currently, the transaction cost to mint a new NFT on Hedera is $0.05. However, the cost to transfer an already minted NFT is only $0.001. This may lead to some developers recycling NFTs to avoid the larger cost of minting a new NFT. In order to incentivize the proper disposal of NFTs that are no longer in use and discourage the practice of recycling NFTs, I propose a discounted transaction fee for minting NFTs.

## Proposed Solution
For every X number of NFTs that are burned, the next X number of NFTs that are minted will have a discounted transaction fee of $0.001, the same as the cost of transferring an NFT(i.e. the effective cost of recycling an NFT which only requires an NFT transfer). This approach not only incentivizes the proper disposal of unused NFTs, but also discourages the practice of recycling NFTs to avoid the larger cost of minting a new NFT. After these X discounted mints have been used, any additional NFT mints would need to be done at the normal NFT mint cost of $0.05.

## Conclusion
By implementing this count-based approach to NFT mint fees, we can promote the responsible disposal of unused NFTs and discourage the practice of NFT recycling.

**Comment by Ashe-Oro** [comment]
Interesting idea.  Do you feel that token rent will not be enough incentive
for responsible NFT management?

**Reply by mshakeg** [reply]
Hi @Ashe-Oro I don't see any fees for [token rent](https://docs.hedera.com/hedera/networks/mainnet/fees). Could you please share where that's documented?

If token rent is based on the number of NFTs issued then it may possibly be cheaper to burn and mint fresh NFTs than recycling NFTs, but only if burning and minting occurs relatively infrequently, some applications require frequent burning and minting of NFTs and so in such applications NFT recycling would probably work out cheaper at current pricing.

**Reply by Ashe-Oro** [reply]
While token rent is something on the roadmap, I don't believe it's been defined yet as far as the actual costs. But that said, I think it will probably be enough to correctly incentivize users and token creators.

**Reply by mshakeg** [reply]
@Ashe-Oro I see, is there a HIP for it? I couldn't find a discussion or HIP searching for it.

**Comment by Ashe-Oro** [comment]
I don't believe there is, which shows you how far in the future it is ;)
For now, I dont think HTS token rental will be enabled any time soon. When
we have a HIP for it, let's reconvene and see if it covers this use case.
